apps/src/p5lab/spritelab/commands/spriteCommands.js

```diff
@@ -173,5 +174,14 @@ export const commands = {
         );
       sprite.scale *= sprite.baseScale;
     });
+  },
+
+  makeNewSpriteGroup(n, animation, layout) {
```

Comment:
What's n in this case? It may clear up some confusion to add a bit more context in the variable name. For example `spriteCount` or `numGroups` etc.

Reply:
We usually use `numSprites` for this argument in similar commands (see `makeNumSprites` or `makeBurst` defined above at lines 68 and 83).

Reply:
👍 can do!